### DISCUSSION TITLE
Adding Custom Web Searches Without Search Query

### DISCUSSION THREAD

**Discussion by robrvera** [Flow-Launcher/Flow.Launcher#3155] [Q&A]
Hello, I am new to Flow Launcher and GitHub (I just made an account to ask this question lol). I am confused on whether a user can add custom web searches, without needing some sort of search query.

For example, at my job, I heavily use Monday.com dashboards and vendor portals through chrome. I would like to just open Flow Launcher, enter a specified keyword, press enter and be taken to a desired website. Is this possible?

**Comment by jjw24** [comment]
Hi there, apologies for the late reply. From what you described it sounds like you just need to bookmark the desired page then hit `b` (this uses the BrowserBookmarks plugin) and select the bookmarked url.

**Reply by robrvera** [reply]
Thanks for this! I was more so talking on something similar to what Listary or Alfred have. Where a user is able to add custom web searches with a hotkey, through the app itself. Is the only way to achieve something like this through the use of having bookmarks on a browser?
